src/test/scala/beam/agentsim/infrastructure/ZonalParkingManagerSpec.scala

```diff
@@ -7,7 +7,8 @@ import akka.testkit.{ImplicitSender, TestActorRef, TestKit}
 import akka.util.Timeout
 import beam.agentsim.Resource.ReleaseParkingStall
 import beam.agentsim.agents.BeamvilleFixtures
-import beam.agentsim.infrastructure.parking.{ParkingType, PricingModel}
+import beam.agentsim.infrastructure.parking.PricingModel.{Block, FlatFee}
```

Comment:
please share results of parking utilization with sflight, e.g. is without change same as with beamConfig.beam.agentsim.taz.parkingNumberOfClusters=1?

also share results as you increase beamConfig.beam.agentsim.taz.parkingNumberOfClusters, e.g. 2,4,8,16,32

Reply:
expected:
- e.g. does parking distance increase due to higher number of clusters?
- do we run out of parking in some parking managers due to the split? e.g. reduce parkingStallCountScalingFactor to provoke this